safetrace-sdk/SafeTrace.swift

```diff
@@ -115,16 +120,14 @@ public final class SafeTrace {
             // If we're launching for bluetooth background activity, start scanning only
             // but do not perform any networking side effects
             environment.tracer.startScanning()
+            sendHealthCheckForBluetoothWakeIfNeeded()
         } else {
             environment.tracer.startScanning()
             environment.traceIDs.refreshIfNeeded()
             environment.tracer.reportPendingTraces()
             
             let isFromRemotePush = launchOptions?[.remoteNotification] != nil
-            let task = UIApplication.shared.beginBackgroundTask()
```

Comment:
Moved the background task handling to the `sendHealthCheck` method so it will run every time